nbs/01a_losses.ipynb

```diff
@@ -62,7 +62,15 @@
     "class BaseLoss():\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.            axis:int=-1, # The axis on which to reduce
</pre><p>Should docment just say <code># reduction axis</code>  to be more concise? </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/muellerzr/fastai-docment-sprint/pull/87/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>